frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx

```diff
@@ -173,6 +202,14 @@ export const VMDisksFirehose: React.FC<VMLikeEntityTabProps> = ({ obj: vmLikeEnt
       namespace,
       prop: 'datavolumes',
     }),
+    getResource(TemplateModel, {
```

Comment:
we know the template name and its namespace. We can load it here and don't need all other templates

Reply:
done.